src/mds/MDSAuthCaps.h

```diff
@@ -226,7 +225,7 @@ class MDSAuthCaps
 
   friend std::ostream &operator<<(std::ostream &out, const MDSAuthCaps &cap);
 private:
-  CephContext *cct = nullptr;
+  CephContext *cct = g_ceph_context;
```

Comment:
If we're going to use `g_ceph_context` (a global variable), why bother storing it in the cap structure?

Reply:
This comment has been taken care of.